src/bootstrap/src/core/config/flags.rs

```diff
@@ -388,6 +388,9 @@ pub enum Subcommand {
         /// enable this to generate a Rustfix coverage file, which is saved in
         /// `/<build_base>/rustfix_missing_coverage.txt`
         rustfix_coverage: bool,
+        #[arg(long, alias("nocapture"))]
+        /// don't capture stdout/stderr of tests
+        no_capture: bool,
```

Comment:
Suggestion: I get confused every time I recall that `--nocapture` is `--nocapture` instead of `--no-capture`. Can we just drop the current libtest naming and stick with `--no-capture`?

Reply:
For the record, renaming this cli arg in libtest to `--no-cature` was done by T-testing-devex in #133073 (thank you epage 💙).